.travis.yml

```diff
@@ -1,9 +0,0 @@
-language: node_js
-node_js:
-- 12
-install:
```

Comment:
I think you need to port this over? maybe .github/workflows/release.yml ?

Reply:
for this particular project it doesn't require build artifacts to be attached to the release.

But yes, some other projects do.